db/osqlrepository.c

```diff
@@ -305,12 +305,11 @@ int osql_repository_rem(osql_sess_t *sess, int lock, const char *func, const cha
                     p, found_uuid);
         }
 
-        abort();
         if (lock) {
```

Comment:
Looks like the abort is meant more like a temporary patch that will allow future investigation in what produce a certain type of error (and probably we didn't figure out how to unwind properly the error).